lib/relation-definition.js

```diff
@@ -1276,8 +1313,12 @@ BelongsTo.prototype.related = function (refresh, params) {
   var scopeQuery = null;
   
   if (arguments.length === 1) {
+    if (typeof refresh === 'boolean') {
+      params = utils.createPromiseCallback();
+    } else {
```

Comment:
Oops.
Yeah, I added the `get()` methods but didn't remove promises from the `related()` methods.
...although it doesn't really hurt anything, I guess...